### DISCUSSION TITLE
[Feature] Display duplicates utility buttons (keep/trash) BEFORE the pictures

### DISCUSSION THREAD

**Discussion by deurk** [immich-app/immich#10450] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

UI wise, putting the keep/trash button bar above the pictures would allow the user to batch delete duplicates without having to move the cursor of the mouse on the desktop version. Gain of time!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by deurk** [comment]
Closed as not approved?

**Reply by bo0tzz** [reply]
Closed as I believe it's been the way you requested for a while now
